files/synchronize.py

```diff
@@ -306,7 +316,7 @@ def main():
     else:
         private_key = '-i '+ private_key 
 
-    ssh_opts = '-S none -o StrictHostKeyChecking=no'
+    ssh_opts = '-S none -o StrictHostKeyChecking=no %s' % ssh_args
```

Comment:
`ssh_args` and `use_ssh_args` are 2 different things.  I think you are missing the other piece which is the action plugin which has a PR at https://github.com/ansible/ansible/pull/10612